lib/system.nim

```diff
@@ -833,7 +835,7 @@ type
     ##
     ## This is only raised if the `segfaults module <segfaults.html>`_ was imported!
 
-when defined(js) or defined(nimdoc):
+when defined(JS) or defined(nimdoc):
```

Comment:
nit: `defined(js)` is the more common convention

Reply:
and i was looking at whole nim repo: 83 vs 50 JS

Reply:
> 83 vs 50 JS

Since this is happening in other places too, I'll make another PR which will convert all `JS` to lower-case, not to pollute this PR with unrelated changes.